.github/workflows/tests.yml

```diff
@@ -19,5 +19,4 @@ jobs:
       - uses: actions/checkout@dcd71f646680f2efd8db4afa5ad64fdcba30e748
       - name: Run tests
         run: make test-exercises
-        # TODO: fix generators
-        #run: make test-generator
+        run: make test-generator
```

Comment:
You can't have two `run` keys. To run multiple commands, the `|` character needs to be used. See https://docs.github.com/en/github-ae@latest/actions/using-workflows/workflow-commands-for-github-actions#example-grouping-log-lines

Reply:
that's my fault for making the comment this way, then